packages/ui/test/bounty/hooks/useBounties.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { renderHook } from '@testing-library/react-hooks'
+import React from 'react'
+
+import { bountyPeriods } from '@/bounty/helpers'
+import { useBounties, UseBountiesProps } from '@/bounty/hooks/useBounties'
+import { asPeriod } from '@/bounty/types/casts'
+import { seedBounties, seedMember } from '@/mocks/data'
+
+import { MockQueryNodeProviders } from '../../_mocks/providers'
+import { setupMockServer } from '../../_mocks/server'
+import { MEMBER_ALICE_DATA } from '../../_mocks/server/seeds'
+
+const renderUseBounties = (props: UseBountiesProps) =>
+  renderHook(() => useBounties(props), {
+    wrapper: ({ children }) => <MockQueryNodeProviders>{children}</MockQueryNodeProviders>,
+  })
+
+describe('useBounties', () => {
+  const mockServer = setupMockServer({ noCleanupAfterEach: true })
+
+  beforeAll(() => {
+    seedMember(MEMBER_ALICE_DATA, mockServer.server)
+    seedBounties(mockServer.server)
```

Comment:
```suggestion
    const bounty = {
      creatorId: undefined,
      oracleId: undefined,
      contractType: { type: 'Open' },
      discussionThreadId:  mockThreads[0].id,
    }
    seedMembers(server.server, 2)
    seedForumCategory(mockCategories[0], mockServer.server)
    seedForumThread(mockThreads[0], mockServer.server)
    seedBounties(mockServer.server, [
      { ...bounty, creatorId: MEMBER_ALICE_DATA },
      { ...bounty, oracleId: MEMBER_ALICE_DATA  },
    ])
```
(See `test/forum/components/PostListItem.test.tsx` for the forum mocking).
There is a slight chance this would work but you probably need to seed the contributions and entries (depending on the query). Maybe a better version of this should go in a helper file in `test/_mocks/bounty/`.

It currently fails because only one member (alice) is seeded but `seedBounties(mockServer.server)` will try to seed all bounties from the generated mocks which refers to any members. However the same problem would appear with `oracleId`, `contractType.whitelistIds`, and `discussionThreadId`.

As written on the testing doc you shouldn't seed every entity (even every members) because that would slow the tests down. So the mocks needs to be modified a bit.